### ISSUE TITLE
[Documentation / Request to add to the FAQ entry] Using --wid option for embedded video player via ruby-gtk3

### ISSUE THREAD

**Issue by rubyFeedback** [mpv-player/mpv#8498]
Hey guys,

I am writing a lot of ruby-gtk3 code lately, the last some weeks.

Today I found out that you can actually "embed" videos into ruby-gtk
applications, by using --wid=.

I saw this on a webforum for mplayer, but of course mpv supports --wid
too, and it works even better than mplayer (mplayer has some strange
blurry background on the top and bottom pane; mpv does not. No idea
but the commandline used was almost identical so I assume mpv beats
mplayer here).

GTK gives back the .xid or .wid value, and then I can use it:

    --wid=69206019

My next idea is to build a minimalistic, really stupid GUI around that,
to allow "embedding". Perhaps I have a separate Gtk::Window for
the main video, and a separate Gtk::Window for the control options.

Actually I don't know yet whether one can control such a mpv instance,
but I assume if --wid works then I could send pause, resume? That
would be quite neat as I could really have a super-simple GUI, not
so much for my own use case (I use the commandline mostly)
but for elderly people and such who need things to be simple.

Anyway - sorry for the lengthy explanation. NOW to the issue here.

As you may guess, I am learning slowly what is all possible, so
mostly trial and error and reading random snippets. :P

It WOULD however had be quite neat if the FAQ of mpv could
have an entry about that, ideally if someone who knows more
than I do about --wid and any limitations. Perhaps pygtk also
has something like that.

For anyone interested in that, or reading this later, or doing
some python code too (the python gtk examples work
almost identical to ruby, if you adjust the different names
and method calls a bit):

    require 'gtk3'
    window = Gtk::Window.new
    window.set_size_request(800, 600)
    window.realize
    xid_value = window.window.xid.to_s
    _ = 'mpv --wid='+xid_value+' --start=280 video_file_goes_in_here.mp4' # Just to demonstrate start-position
    puts _
    system

So, TL; DR - please consider adding a short subsection to the FAQ about the --wid option,
in particular caveats but perhaps also other applications that make use of it (that way I
could peek at the source code and see how they do it; I had a look at gstreamer but
this is way over my head in its complexity...)

**Comment by Akemi** [comment]
i am not sure what you actually expect. we have an explanation about the `--wid` option here https://github.com/mpv-player/mpv/blob/d5ab5482a9b35650600ca68bcfee752995e8337b/DOCS/man/options.rst.

we have exmaple apps, that also use wid, here https://github.com/mpv-player/mpv-examples

if youw ant to control mpv use the json ipc https://github.com/mpv-player/mpv/blob/master/DOCS/man/ipc.rst or more advanced libmpv https://github.com/mpv-player/mpv/blob/master/DOCS/man/libmpv.rst

i don't think anyone from us is interested in documenting the `--wid` option any further, atm. though feel free to open a PR or add a new section to our wiki https://github.com/mpv-player/mpv/wiki.
